doc/dev-guide/src/linting.md

```diff
@@ -0,0 +1,31 @@
+# Linting
+
+## Manual linting
+
+When checking the codebase with [`clippy`](https://doc.rust-lang.org/stable/clippy/index.html), it is recommended to use:
+
+`cargo +beta clippy --all --all-targets --all-features -- -D warnings`
```

Comment:
``````suggestion
```console
$ cargo +beta clippy --all --all-targets --all-features -- -D warnings
```
``````

Reply:
oh nice thanks, i indeed thought about this not looking the best it could! Thanks for the solution!